mod_ci/controllers.py

```diff
@@ -99,38 +100,28 @@ def start_platforms(db, repository, delay=None, platform=None) -> None:
 
     with app.app_context():
         from flask import current_app
+        app = current_app._get_current_object()
         if platform is None or platform == TestPlatform.linux:
-            linux_kvm_name = config.get('KVM_LINUX_NAME', '')
             log.info('Define process to run Linux VM')
-            linux_process = Process(target=kvm_processor, args=(current_app._get_current_object(), db, linux_kvm_name,
-                                                                TestPlatform.linux, repository, delay,))
+            linux_process = Process(target=gcp_instance, args=(app, db, TestPlatform.linux, repository, delay))
             linux_process.start()
             log.info('Linux VM process kicked off')
 
         if platform is None or platform == TestPlatform.windows:
-            win_kvm_name = config.get('KVM_WINDOWS_NAME', '')
             log.info('Define process to run Windows VM')
-            windows_process = Process(target=kvm_processor, args=(current_app._get_current_object(), db, win_kvm_name,
-                                                                  TestPlatform.windows, repository, delay,))
+            windows_process = Process(target=gcp_instance, args=(app, db, TestPlatform.windows, repository, delay))
             windows_process.start()
             log.info('Windows VM process kicked off')
 
 
-def kvm_processor(app, db, kvm_name, platform, repository, delay) -> None:
+def gcp_instance(app, db, platform, repository, delay) -> None:
     """
-    Check whether there is no already running same kvm.
-
-    Checks whether machine is in maintenance mode or not
-    Launch kvm if not used by any other test
-    Creates testing xml files to test the change in main repo.
-    Downloads the build artifacts generated during GitHub Action workflows.
+    Find all the pending tests and start running them in new GCP instances.
```

Comment:
The docstring sounds like we are using one gcp instance for each test, is that the case?

Reply:
Yes for one PR/commit there is 1 VM instance, and all regression tests for that test are ran on the same instance.